<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>home</class>
 <widget class="QMainWindow" name="home">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>778</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cipher Tools</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resource.qrc">
    <normaloff>:/Resource/CipherTools_Icons.svg</normaloff>:/Resource/CipherTools_Icons.svg</iconset>
  </property>
  <widget class="QWidget" name="Home">
   <widget class="QPushButton" name="hidemode">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>86</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>私隐模式</string>
    </property>
   </widget>
   <widget class="QPushButton" name="settings">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>310</y>
      <width>86</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>设定</string>
    </property>
   </widget>
   <widget class="QLabel" name="hostname">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>HOSTNAME</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>351</width>
      <height>58</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="v4">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>公网 IPv4 地址</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="v4add">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Loading......</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>351</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="v6">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>公网 IPv6 地址</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="v6add">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Loading......</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>351</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_4">
     <item row="0" column="0">
      <widget class="QLabel" name="isp">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ISP</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="ispinfo">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Loading......</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>351</width>
      <height>58</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="localv4">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>本地 IPv4 地址</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="localv4add">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Loading......</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>100</y>
      <width>351</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_5">
     <item row="0" column="0">
      <widget class="QLabel" name="localv6">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>本地 IPv6 地址</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="localv6add">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Loading......</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>180</y>
      <width>351</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_6">
     <item row="1" column="0">
      <widget class="QLabel" name="Mac">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Loading......</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="currentMac">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>当前网卡 MAC 地址</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="refresh">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>310</y>
      <width>86</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>刷新</string>
    </property>
   </widget>
   <widget class="QLabel" name="priority">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>PRIORITY</string>
    </property>
   </widget>
   <widget class="QLabel" name="test_waring">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>370</y>
      <width>521</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>6</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TEST_VERSION | WARNING</string>
    </property>
    <property name="textFormat">
     <enum>Qt::TextFormat::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Version">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>350</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>VERSION NAME</string>
    </property>
    <property name="textFormat">
     <enum>Qt::TextFormat::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>778</width>
     <height>24</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="helpcenter">
    <property name="title">
     <string>帮助中心</string>
    </property>
    <widget class="QMenu" name="repos">
     <property name="title">
      <string>查看源码</string>
     </property>
     <addaction name="repoCNB"/>
     <addaction name="repoGithub"/>
     <addaction name="repoGitee"/>
     <addaction name="repoCodeberg"/>
    </widget>
    <widget class="QMenu" name="issue">
     <property name="title">
      <string>问题反馈</string>
     </property>
     <addaction name="issueCNB"/>
     <addaction name="issueGithub"/>
     <addaction name="issueCodeberg"/>
     <addaction name="issueGitee"/>
    </widget>
    <widget class="QMenu" name="log">
     <property name="title">
      <string>更新日志</string>
     </property>
     <addaction name="logGithub"/>
     <addaction name="logGitee"/>
     <addaction name="logCNB"/>
     <addaction name="logCodeberg"/>
    </widget>
    <addaction name="about"/>
    <addaction name="wiki"/>
    <addaction name="separator"/>
    <addaction name="blog"/>
    <addaction name="repos"/>
    <addaction name="log"/>
    <addaction name="issue"/>
   </widget>
   <widget class="QMenu" name="tools">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>工具</string>
    </property>
    <widget class="QMenu" name="Ping">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>延迟检测</string>
     </property>
     <addaction name="PingCS"/>
     <addaction name="PINGGNT"/>
    </widget>
    <addaction name="MOWeb"/>
    <addaction name="deviceinfo"/>
    <addaction name="MOQt"/>
    <addaction name="WOL"/>
    <addaction name="track"/>
    <addaction name="NAT"/>
    <addaction name="Ping"/>
   </widget>
   <addaction name="tools"/>
   <addaction name="helpcenter"/>
  </widget>
  <action name="WOL">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>局域网唤醒</string>
   </property>
  </action>
  <action name="track">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>路由检测</string>
   </property>
  </action>
  <action name="NAT">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>NAT 检测</string>
   </property>
  </action>
  <action name="about">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="wiki">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Wiki（在建）</string>
   </property>
  </action>
  <action name="PingCS">
   <property name="text">
    <string>PingCS 《反恐精英》延迟检测</string>
   </property>
  </action>
  <action name="PINGGNT">
   <property name="text">
    <string>全球网测延迟检测</string>
   </property>
  </action>
  <action name="PingVAL">
   <property name="text">
    <string>PingVAL《无畏契约》国服延迟检测</string>
   </property>
  </action>
  <action name="repoCNB">
   <property name="text">
    <string>CNB</string>
   </property>
  </action>
  <action name="repoGithub">
   <property name="text">
    <string>Github</string>
   </property>
  </action>
  <action name="issueCNB">
   <property name="text">
    <string>CNB</string>
   </property>
  </action>
  <action name="issueGithub">
   <property name="text">
    <string>Github</string>
   </property>
  </action>
  <action name="blog">
   <property name="text">
    <string>博客</string>
   </property>
  </action>
  <action name="aboutqt">
   <property name="text">
    <string>关于Qt</string>
   </property>
  </action>
  <action name="action">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>设备信息</string>
   </property>
  </action>
  <action name="MOWeb">
   <property name="text">
    <string>多出口（基于 Yumeyo 网络综合查询器多出口 Web 版）</string>
   </property>
  </action>
  <action name="deviceinfo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>设备信息</string>
   </property>
  </action>
  <action name="MOQt">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>多出口（非浏览器版本）</string>
   </property>
  </action>
  <action name="logGithub">
   <property name="text">
    <string>GitHub</string>
   </property>
  </action>
  <action name="logGitee">
   <property name="text">
    <string>Gitee</string>
   </property>
  </action>
  <action name="logCNB">
   <property name="text">
    <string>CNB</string>
   </property>
  </action>
  <action name="repoGitee">
   <property name="text">
    <string>Gitee</string>
   </property>
  </action>
  <action name="logCodeberg">
   <property name="text">
    <string>Codeberg</string>
   </property>
  </action>
  <action name="issueCodeberg">
   <property name="text">
    <string>Codeberg</string>
   </property>
  </action>
  <action name="repoCodeberg">
   <property name="text">
    <string>Codeberg</string>
   </property>
  </action>
  <action name="issueGitee">
   <property name="text">
    <string>Gitee</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections/>
</ui>
